src/ui/Card/index.tsx

```diff
@@ -65,10 +67,14 @@ const SecondaryCard = ({
   footer,
 }: CardProps) => (
   <CardContainer theme={theme} variant="secondary">
-    <CardHeader theme="dark" center={true}>
-      <CardTitle theme="dark" text={title.text} tag="h2" size="primary" />
-      <time className={styles.date}>{date}</time>
-    </CardHeader>
+    {(title || date) && (
+      <CardHeader theme="dark" center={true}>
+        {title && (
+          <CardTitle theme="dark" text={title.text} tag="h2" size="primary" />
+        )}
+        <time className={styles.date}>{date}</time>
```

Comment:
Why not just {date && <time .... { i.s.o the code on ln 70?

Reply:
To prevent the rendering of the CardHeader container component when only a date is available.